### DISCUSSION TITLE
cannot download from miktex website

### DISCUSSION THREAD

**Discussion by 10UAikawa** [MiKTeX/miktex#1407] [Q&A]
Hi all is there any way to get miktex because the website download page is down because it won't let me download miktex

**Comment by tsotnek** [comment]
I confirm, it has been down for the whole day.

**Comment by jwiduch** [comment]
I can't update packages using MiKTeX console. I receive "Error response from server: 500". Is this error due to problems with the website?

**Comment by twistxj** [comment]
ive tried to update for the past few days and it says the server is down.

**Comment by gamerhawk** [comment]
Hi, I am also not able to download it from download page.it is also not updating in MiKTeX console.
